src/main/bash/commands/kobman-install.sh

```diff
@@ -111,10 +59,35 @@ function __kobman_validate_version_format
 
 function __kobman_check_and_confirm_existing_version
 {
-       	git ls-remote --tags "https://github.com/${1}/KOBman" | grep -w 'refs/tags/[0-9]*\.[0-9]*\.[0-9]*' | sort -r | head | grep -o '[^\/]*$' | grep -w "$2" > /dev/null
+	env=$1
+	version=$2
+	namespace=$3	
+	__kobman_environment_name_align "$env"
 
+	git ls-remote --tags "https://github.com/${namespace}/${confirmed_environment}" | grep -w 'refs/tags/[0-9]*\.[0-9]*\.[0-9]*' | sort -r | head | grep -o '[^\/]*$' | grep -w "${version}" > /dev/null
```

Comment:
This will not work in offline mode.

Reply:
should be using list.txt for offline mode...right?

Reply:
@asa1997  @EtricKombat list.txt must be used for both modes, you need to only check if list.txt is different from what you have on the host. 
please note bare kobman should not have any dependency on git.

Reply:
so instead of using git ls-remote to check the remote tagged version , we have to check the file list.txt in the local system using grep for the available version/validity for offline mode as well as online mode.

Reply:
@EtricKombat  @asa1997  yes

Reply:
What should be the other uses of this online and offline mode.

Reply:
I mean functionalities

Reply:
The general idea behind  offline mode in kobman is to use kobman without active internet connectivity ,i.e using cached resources(zip or software bundles) required to install different kobman envirornment scripts.

For e.g. In offline mode some change that should be anticipate later are:  
a) The list file downloaded  on the host and made available @ .kobman/var/list.txt will be used, by the kob list command. However when we switch to online mode it will check if the .kobman/var/list.txt is same as the one in the dist branch .
b)Similarly the kobman environment scripts should also support offline mode. i.e when offline mode is enabled you can also re-install  of environment from cached  kobman resources associated with the kob environment (i.e already downloaded installable zip/archive file for that environment).

Reply:
@sumodgeorge  @asa1997 Please do include Support for offline mode to KOBman features set and have a RF number for it we can mention here.